edr_plugin/gui/ui/query_corridor.ui

```diff
@@ -230,12 +247,12 @@
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
-       <weight>75</weight>
-       <bold>true</bold>
+       <weight>50</weight>
+       <bold>false</bold>
       </font>
      </property>
      <property name="text">
-      <string>Geometry WKT:</string>
+      <string>Or Enter Geometry WKT (as avalid LineString / LineStringZ / LineStringM / LineStringZM) </string>
```

Comment:
This needs to change to be Or Enter Geometry WKT (as **a valid** LineString / LineStringZ / LineStringM / LineStringZM)